predicthq/endpoints/schemas.py

```diff
@@ -126,10 +124,10 @@ def _normalize(self, value):
 
 class Area(StringModel):
 
-    import_format = r"(?P<radius>\d+(k?m|ft|mi))@(?P<latitude>-?\d+(\.\d+)?),(?P<longitude>-?\d+(\.\d+)?)"
+    import_format = r"(?P<radius>(\d+(\.\d+)?)(km|mi)|(\d+)(m|ft))@(?P<latitude>[\-\+]?\d+(\.\d+)?),(?P<longitude>[\-\+]?\d+(\.\d+)?)"
     export_format = "{radius}@{latitude},{longitude}"
 
-    radius = StringType(regex=r"\d+(k?m|ft|mi)", required=True)
+    radius = StringType(regex=r"(\d+(\.\d+)?)+(km|mi)|(\d+)+(m|ft)", required=True)
```

Comment:
You've got some extra `+` here?

Reply:
I am not sure that you got your parentheses right.

Reply:
ok, it doesnt seme to affects the actual regex after trying it a bit, but it better to wrap it just incase.